llap-tez/src/java/org/apache/hadoop/hive/llap/tezplugins/LlapTaskSchedulerService.java

```diff
@@ -1820,6 +1830,8 @@ private static boolean removeFromRunningTaskMap(TreeMap<Integer, TreeSet<TaskInf
               hostList.add(nodeInfo);
             }
           }
+        } else {
+          LOG.warn("Null NodeInfo when attempting to get available resources");
```

Comment:
Maybe also add workerIdentity in the warn message?

Reply:
done